### DISCUSSION TITLE
Probleme bei Umsetzung von Phoniebox mit Pirate Audio

### DISCUSSION THREAD

**Discussion by kkmichi** [MiczFlor/RPi-Jukebox-RFID#1368] [Q&A]
Hallo liebe Community,
ich habe den Raspberry Pi Zero WH, den ofoff-shim sowie den Pirate Audio Dac mit Display. Ich bin langjähriger Linux-Anwender, jedoch ohne Programmiererfahrung, etc.
Mein Ziel ist eine Phoniebox mit dem Pirate Audio zu realisieren. Ich bin nach mehreren Versuchen jetzt am Ende meiner Weisheit. Ich habe das Gefühl, dass Phoniebox (Classic) und Pirate Audio nicht "zusammenarbeiten". Das Display zeigt immer nur "Visit http://192.168... to select content", die Lautstärketasten ändern zwar den Balken im Display, aber die Steuerung über MPD/Phoniebox ist nicht "synchron".

Mein Vorgehen war zuerst raspbian lite, dann onoff shim, dann das Installationsskript der Phoniebox, dann das Installationsskript von Pirate Audio. Falls jemand dies bereits so umgesetzt hat: welche Auswahlmöglichkeiten soll ich im Skript der Phoniebox auswählen? Die Hinweise von Splittscheid, https://github.com/pimoroni/pirate-audio/tree/master/mopidy, https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/components/audio/PirateAudioHAT/README.md) habe ich beachtet. Danke für Hilfe und Hinweise.

**Comment by s-martin** [comment]
See answer in #1369
